doc/python/selections.md

```diff
@@ -0,0 +1,123 @@
+---
+jupyter:
+  jupytext:
+    notebook_metadata_filter: all
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.7
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+  language_info:
+    codemirror_mode:
+      name: ipython
+      version: 3
+    file_extension: .py
+    mimetype: text/x-python
+    name: python
+    nbconvert_exporter: python
+    pygments_lexer: ipython3
+    version: 3.9.0
+  plotly:
+    description: How to use selections in Python. Examples of adding and styling selections.
+    display_as: file_settings
+    language: python
+    layout: base
+    name: Selections
+    order: 26
+    permalink: python/selections/
+    thumbnail: thumbnail/make_selection.jpg
+---
+
+## Adding Selections
+
+*New in 5.10*
+
+You can add persistent selections to a rendered figure using the **Box Select** and **Lasso Select** tools in the mode bar.
+To add multiple selections, select **Shift** when making new selections.
+To clear a selection, double-click it. You can clear all selections by double-clicking any unselected area of the figure.
+
+
+
+You can also add selections to a figure that displays when it renders using `fig.add_selection`.
+Here, we add a rectangular selection that connects the points `3.0`, `6.5`, `3.5`, and `5.5`.
+
+
+```python
+import plotly.express as px
+
+df = px.data.iris()
+
+fig = px.scatter(df, x="sepal_width", y="sepal_length")
+fig.add_selection(type="rect", x0=3.0, y0=6.5, x1=3.5, y1=5.5)
+
+fig.show()
+```
+
```

Comment:
The `path` option should be described here as well.

Reply:
Thanks @archmoj. Trying to add this now. Should this render something?
It is the same as the earlier example, but just uses path.
Nothing shows on the figure with this.

```
import plotly.express as px

df = px.data.iris()

fig = px.scatter(df, x="sepal_width", y="sepal_length")
fig.add_selection(type="path", path="M2000,65L4000,75L8000,65Z")

fig.show()

```

Reply:
It should. What happens if you draw a `shape` instead?
```py
fig.add_shape(type="path", path="M2000,65L4000,75L8000,65Z")
```

Reply:
Yeah, this one works. 
So actually it seems like the selection is only immediately visible in the range of the existing data.

So, this works:

```

df = px.data.iris()

fig = px.scatter(df, x="sepal_width", y="sepal_length")
fig.add_selection(type="path", path="M2,6L4,7L4,6Z")
#fig.add_selection(type="path", path="M20,6L4000,7L8,65Z")

fig.show()
```

<img width="690" alt="image" src="https://user-images.githubusercontent.com/12749831/180281338-55ebdfe4-c44c-4fd5-b5a6-eb81535f3108.png">


Added an example now.